final_task/rss_reader/converter.py

```diff
@@ -0,0 +1,153 @@
+"""This module converts data to pdf"""
+
+from fpdf import FPDF
+import urllib.request as url
+import os
+import sys
+import datetime
+
+this_directory = os.path.abspath(os.path.dirname(__file__))
+font = 'ARIALUNI.TTF'
```

Comment:
Global constants usually declared in uppercase e.g. `THIS_DIRECTORY`

Reply:
All global variables were fixed, it will be updated during the next commit.